### ISSUE TITLE
Tile loading error

### ISSUE THREAD

**Issue by Saturn87930** [Mbucari/GEHistoricalImagery#2]
Why doesn't it work when I download the tiles locally and try to load them with Leaflet?

**Comment by Mbucari** [comment]
What's not working? Do the tiles download? Are the jpegs viewable an image viewer?

**Comment by Saturn87930** [comment]
Yes, the picture can be downloaded and opened with a picture viewer. However, when I load local tiles with leaflet, the order is wrong. Is the row and column number of the picture incompatible with leaflet?

**Comment by Mbucari** [comment]
When you say you're trying to load the local tiles, are you referring to the jpeg tiles that are downloaded with the `dump` command, or the tiles that are in the cache folder? 

Also, I have never worked with leaflet. I have no idea how that's supposed to work or interoperate with imagery.

**Comment by cynomops** [comment]
PS C:\GEHistoricalImagery-v0.1.7-win64> .\GEHistoricalImagery info --location 39.630575,-104.841299 --zoom 21
Invalid location coordinate.
 Location must be in decimal Lat,Long. e.g. 37.58289,-106.52305
 
 C:\GEHistoricalImagery-v0.1.7-win64>GEHistoricalImagery info --location 39.630575,-104.841299
Invalid location coordinate.
 Location must be in decimal Lat,Long. e.g. 37.58289,-106.52305

**Comment by Mbucari** [comment]
@cynomops Very strange. I just downloaded the 1.7 release and tested it with your command and it works.

[](https://github.com/user-attachments/assets/d3a5e1ef-399c-466b-afa0-dbc1c0689159)

Try this instead:
```Console
.\GEHistoricalImagery info --location=39.630575,-104.841299 --zoom=21
```
Did that work?

**Comment by Mbucari** [comment]
[Here's a little more info about the tile numbering and conversion to lat-long.](
https://github.com/Mbucari/GEHistoricalImagery/blob/master/docs/dump.md#convert-between-latlong-and-rowcolumn-numbers)

**Comment by Mbucari** [comment]
@cynomops 

I figured out what was causing this error. I'm sure it's the same bug as #6, and it's fixed in [the latest release](https://github.com/Mbucari/GEHistoricalImagery/releases/latest).
